modules/agent-install-mirroring-for-disconnected-registry.adoc

```diff
@@ -0,0 +1,253 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// * installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc
+// re-use of applicable content from disconnected install mirroring
+
+:_content-type: PROCEDURE
+[id="agent-install-mirroring-for-disconnected-registry_{context}"]
+= Mirroring the {product-title} image repository for a disconnected registry
+
+Mirror the {product-title} image repository to your registry to use during cluster installation.
+
+.Procedure
+
+. Review the
+link:https://access.redhat.com/downloads/content/290/[{product-title} downloads page]
+to determine the version of {product-title} that you want to install and determine the corresponding tag on the link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Repository Tags] page.
+
+. Set the required environment variables:
+.. Export the release version:
++
+[source,terminal]
+----
+$ OCP_RELEASE=<release_version>
```

Comment:
Should this command (and the following) have export in front, e.g.:
$ export OCP_RELEASE=<release_version>

Reply:
yes correct, I have rectified all applicable commands